Source/d_main.c

```diff
@@ -1497,8 +1497,8 @@ void D_DoomMain(void)
       mobjinfo[MT_SKULL].deathstate   = S_BSKUL_DIE1;
       mobjinfo[MT_SKULL].damage       = 1;
     }
-  else
-    mobjinfo[MT_SCEPTRE].doomednum = mobjinfo[MT_BIBLE].doomednum = -1;
+  // else
```

Comment:
Could you please comment this out with an `if 0 ... endif` range and add a comment?